### ISSUE TITLE
Can't get this extension working on my jupyter lab environment

### ISSUE THREAD

**Issue by roytseng-tw** [jupyterlab-contrib/jupyterlab-vim#6]
I'm using python 3.7.6
And below is my jupyter enviroment (obtained with `conda list jupyter`)
```
# Name                    Version                   Build  Channel
jupyter                   1.0.0                      py_2    conda-forge
jupyter_client            6.1.3                      py_0    conda-forge
jupyter_console           6.1.0                      py_1    conda-forge
jupyter_core              4.6.3            py37hc8dfbb8_1    conda-forge
jupyterlab                2.1.2                      py_0    conda-forge
jupyterlab_server         1.1.3                      py_0    conda-forge
```
I have tried `jupyter lab clean` and then `jupyter lab build`, but still can't get it working.
I have successfully used the vim extension on jupyterlab 1.X.
I also noticed that there's a system default shortcut `ii` for interrupting the kernel. However, disabling it won't help either.

What's other possible causes ? Thank you very much.

**Comment by axelfahy** [comment]
Maybe you can try to re-create your environment from scratch. 
How did you install the plugin? Using the Extension Manager inside JupyterLab?

**Comment by ricecooker** [comment]
I was having trouble too with Jupyter 2.1.5. To check my extensions, I did the following:
```
$ jupyter labextension list
JupyterLab v2.1.5
Known labextensions:
   app dir: /Users/dcheng/miniconda/share/jupyter/lab
        @axlair/jupyterlab_vim v0.12.4  enabled  OK
        @jupyterlab/toc v4.0.0  enabled  OK
        jupyterlab-plotly v4.8.2  enabled  OK

Uninstalled core extensions:
    @axlair/jupyterlab_vim
```
In order to fix my problem I had to delete the file mentioned in #12. In my case it was `rm ~/miniconda/share/jupyter/lab/settings/*.json`

After that I did a `jupyter lab clean; jupyter lab build`, everything worked.

Hope that helps.

**Comment by StanczakDominik** [comment]
Yup, that's a solid and working fix.

**Comment by oleksii-haponiuk** [comment]
Not sure if my problem is related to this issue, I'm a new user and I don't have old packages installed. Here's my `jupyter labextension list`:
```
JupyterLab v2.1.5
Known labextensions:
   app dir: /Users/Alexei/anaconda3/share/jupyter/lab
        @axlair/jupyterlab_vim v0.12.4  enabled  OK
(base)
```
And it simply doesn't work.

**Comment by arnelg** [comment]
I was not able to make this extension work with jupyterlab 3.0.0 and after installing the extension using `pip install jupyterlab_vim`. I checked that the extension was activated ("enabled OK") using `jupyter labextension list`. I tried the above suggested fix, but it did not help. I then uninstalled the extension again using `pip`, downgraded to jupyterlab 2.2.9 and re-installed the package using `jupyter labextension install @axlair/jupyterlab_vim`. Now it works.

One thing I found confusing: While the extension was _not_ working I was unsure if there was a problem with the installation or if I just didn't know how to enter vim command mode. It would be great if the readme could have something about what it's supposed to look like when it's working.
